src/js/pages/Service.js

```diff
@@ -29,11 +29,15 @@ const i18nMessages = defineMessages({
 });
 
 const Service = ({ service, intl }) => {
-  const { image, title, slug, topic, steps, dynamicContent, additionalContent, contacts, related } = service;
+  const {
+    image, title, slug, topic, steps, dynamicContent, additionalContent,
+    contacts, related
+  } = service;
+
+  const { theme } = topic;
```

Comment:
will a topic only have one parent theme? I forget the relationships. Maybe a service can have multiple topics?